bastion/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.9'
+services:
+  ollama:
+    image: ollama/ollama:0.1.27
+    restart: unless-stopped
+    networks:
+      - bridge
+    ports:
+      - 11434:11434
+    volumes:
+      - /opt/models:/root/.ollama
+
+networks:
+  bridge:
+    name: bridge_network
```

Comment:
you shouldn't need to specify the network, bridge is the default.

Reply:
This is not using the default bridge network but rather the user-defined bridge. The default bridge network does not have automatic DNS resolution between containers. There are other components running on that VM that is not specific to the project, hence the missing context.